extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -309,6 +310,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @private
    */
   findOrBuildCarousel_(lightboxGroupId) {
+    triggerAnalyticsEvent(this.element, 'lightboxOpened', {});
```

Comment:
I don't know how I missed this earlier, but could you put this in `open_()`? There's technically nothing wrong with this, but logically I think you want to trigger this in the last `then` block of the `open_` function, along with `showControls_`.